tachyon/zk/plonk/examples/shuffle_circuit_test.cc

```diff
@@ -251,10 +252,23 @@ constexpr const char* shuffled_tables[2][W][H] = {
 };
 // clang-format on
 
+using GWC = GWCExtension<math::bn254::BN254Curve, kMaxDegree,
+                         kMaxExtendedDegree, math::bn254::G1AffinePoint>;
+using SHPlonk =
+    SHPlonkExtension<math::bn254::BN254Curve, kMaxDegree, kMaxExtendedDegree,
+                     math::bn254::G1AffinePoint>;
+
+using Poly = typename SHPlonk::Poly;
+using Evals = typename SHPlonk::Evals;
+using Commitment = typename SHPlonk::Commitment;
+
+using _LS = typename lookup::halo2::Scheme<Poly, Evals, Commitment>;
+
 template <typename PCS>
-class ShuffleCircuitTest : public CircuitTest<PCS> {
+class ShuffleCircuitTest : public CircuitTest<PCS, _LS> {
```

Comment:
```suggestion
template <typename PCS, typename LS>
class ShuffleCircuitTest : public CircuitTest<PCS, LS> {
```
This modification seems like it would be beneficial for testing different types of Lookups together later on.